openedx/stanford/lms/envs/aws.py

```diff
@@ -1,6 +1,9 @@
 from lms.envs.aws import *
 
 
+CELERY_QUEUES.update({
```

Comment:
Do we actually need to set anything again in `aws.py` if we're not actually pulling values from `json`?
It looks like we're just resetting it to the same value from `common.py` again?

Reply:
That's probably correct for now. I'll revert aws.